npm/src/typings.ts

```diff
@@ -80,6 +81,7 @@ export interface OAuthReqBody {
   code_challenge_method: 'plain' | 'S256' | '';
   provider: 'saml';
   idp_hint?: string;
+  overrideForceAuthn?: boolean;
```

Comment:
shouldn't this just be `forceAuthn`? Why call it overrideForceAuthn?

Reply:
This is to override the forceAuthn flag

Reply:
But we are not overriding it, forceAuthn is an option. Flipping the bool is confusing.